types/heremaps/index.d.ts

```diff
@@ -1739,11 +1777,13 @@ declare namespace H {
              * @property size {H.math.ISize=} - The icon's size in pixel, default is the bitmap's natural size
              * @property anchor {H.math.IPoint=} - The anchorage point in pixel, default is bottom-center
              * @property hitArea {H.map.HitArea=} - The area to use for hit detection, default is the whole rectangular area
-             * @property asCanvas {H.map.HitArea=} - Indicates whether a non canvas bitmap is converted into a canvas, default is true. The conversion improves the rendering performance but it could also cause a higher memory consumption.
-             * @property crossOrigin {boolean} - Specifies whether to use anonynous Cross-Origin Resource Sharing (CORS) when fetching an image to prevent resulting canvas from tainting, default is false. The option is ignored by IE9-10.
+             * @property asCanvas {H.map.HitArea=} - Indicates whether a non canvas bitmap is converted into a canvas, default is true. The conversion improves the rendering performance but it could
+             * also cause a higher memory consumption.
+             * @property crossOrigin {boolean} - Specifies whether to use anonynous Cross-Origin Resource Sharing (CORS) when fetching an image to prevent resulting canvas from tainting, default is
+             * false. The option is ignored by IE9-10.
              */
-            export interface Options {
-                size?: H.math.ISize;
+            interface Options {
+                size?: H.math.ISize | number;
```

Comment:
Yes, see the examples and parameters for `static createIcon()` here: https://developer.here.com/visualization/documentation/datalens/h-datalens-objectlayer.html
Where `options.size: { H.math.ISize | number } [optional]`